webviz_subsurface/ert_jobs/smry2arrow_batch.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+"""Batch conversion of UNSMRY files to Apache Arrow IPC file format
+"""
+
+import argparse
+import glob
+import logging
+import os
+from pathlib import Path
+
+from .smry2arrow import smry2arrow
+
+logger = logging.getLogger(__name__)
+
+
+def _get_parser() -> argparse.ArgumentParser:
+    """Setup parser for command line options"""
+
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+    )
+
+    parser.description = (
+        "Batch conversion of UNSMRY files to Apache Arrow IPC file format.\n"
+        "\n"
+        "Note that if the enspath argument includes wildcards you will have to enclose it\n"
+        "in quotes to stop the shell from expanding it, e.g.:\n"
+        '  smry2arrow_batch "my_folder/realization-*/iter-0"'
+    )
+
+    parser.add_argument(
+        "enspath",
+        type=Path,
+        help="Path with wildcards giving file system location of the ensemble's realizations",
+    )
+    parser.add_argument(
+        "--eclbase",
+        type=Path,
+        help="Eclipse base name",
+        default=Path("eclipse/model/*.UNSMRY"),
+    )
+    return parser
+
+
+def _batch_convert_smry2arrow(
+    ens_path: Path, ecl_base: Path, relative_output_dir: Path
+) -> None:
+    """Does batch conversion of UNSMRY files for all realizations within an ensemble."""
+    globbed_real_dirs = sorted(glob.glob(str(ens_path)))
+
+    for real_dir in globbed_real_dirs:
+        glob_expr = os.path.join(real_dir, ecl_base)
+        globbed_smry_files = sorted(glob.glob(glob_expr))
+        if globbed_smry_files:
+            real_output_dir = os.path.join(real_dir, relative_output_dir)
+            os.makedirs(real_output_dir, exist_ok=True)
```

Comment:
And again if staying with `pathlib.Path`. But again, no real effect 🙈
```suggestion
            real_output_dir.mkdir(parents=True, exist_ok=True)
```